lang/cpp.md

```diff
@@ -0,0 +1,17 @@
+---
+lang: C++
+syntax: cpp
+title: C++ is the most powerful language!
+author: Panquesito7
+---
+
+```c++
+#include <iostream>
+
+int main(){
+    std::cout << "Hello world!\n";
+    return 0;
+}
+```
+
+# C++ is the most powerful language!
```

Comment:
```suggestion
# C++ is the most powerful language!
```
One has to write the whole installation process reg. Compiler, IDE, & extensions of VSC (Visual Studio Code), I am making one for reference, so give me some time.. : )